src/server/trace/viewer/traceViewer.ts

```diff
@@ -140,7 +140,7 @@ export class TraceViewer {
     // Null means no installation and no channels found.
     let channel = null;
     if (traceViewerBrowser === 'chromium') {
-      if (canAccessFile(registry.executablePath('chromium')!)) {
```

Comment:
something not fully related to the PR. canAccessFile is also true if the input argument is a folder path which leads to wrong error messages when a user passes a folder as an executable path.